gdx/res/com/badlogic/gdx/utils/JsonReader.rl

```diff
@@ -1,5 +1,5 @@
 // Do not edit this file! Generated by Ragel.
-// Ragel.exe -G2 -J -o JsonReader.java JsonReader.rl

```

Comment:
What did the `-G2` flag do before? It isn't needed now?

Reply:
`-G2` is for `Really fast goto-driven FSM` but only applies to C and D languages, it's never done anything for Java.